biahub/settings.py

```diff
@@ -23,6 +23,26 @@ class MyBaseModel(BaseModel):
     model_config = ConfigDict(extra="forbid")
 
 
+class ProcessingFunctions(MyBaseModel):
+    function: str
+    input_channels: Optional[List[str]] = None  # Optional
+    kwargs: Dict[str, Any] = {}
+    per_timepoint: bool = True
+
+
+class ProcessingInputChannel(MyBaseModel):
+    path: Union[str, List[str], None]
+    channels: Dict[str, List[ProcessingFunctions]]
+
+
+class TrackingSettings(MyBaseModel):
+    target_channel: str = "nuclei_prediction"
+    mode: Literal["2D", "3D"] = "2D"
+    z_slices: Optional[Tuple[int, int]] = None
```

Comment:
I suggest calling this `z_range` (e.g. (0, 10)) to avoid confusion with slice objects created by `resolve_z_slice` later which would return `slice(0, 10)`